nextflow_schema.json

```
@@ -237,73 +237,85 @@
-                    "maximum": 1.0,
-                    "hidden": true
+                    "minimum": 0,
+                    "maximum": 1,
+                    "hidden": true,
+                    "fa_icon": "fas fa-cogs"
                 },
                 "mhcflurry_mhcnuggets_score_threshold": {
                     "type": "number",
                     "default": 0.426,
-                    "description": "Threshold for binder/non-binder calling when using MHCflurry or MHCnuggets epitope prediction methods. The default value of 0.426 corresponds to an IC50 of ≤500.",
-                    "minimum": 0.0,
-                    "maximum": 1.0,
-                    "hidden": true
+                    "description": "Threshold for binder/non-binder calling when using MHCflurry or MHCnuggets epitope prediction methods. The default value of 0.426 corresponds to an IC50 of \u2264500.",
+                    "minimum": 0,
+                    "maximum": 1,
+                    "hidden": true,
+                    "fa_icon": "fas fa-cogs"
                 },
                 "show_supported_models": {
                     "type": "boolean",
                     "description": "Display supported alleles of all prediction methods and exit.",
                     "fa_icon": "fas fa-question-circle"
                 },
-                "proc_chunk_size": {
-                    "type": "integer",
-                    "default": 7500000,
-                    "description": "Limits the number of peptides processed at once in the SPLIT_PRED_TASKS process and thus the memory footprint of the process. This parameter does not affect the pred_chunk_size parameter."
-                },
-                "ds_prep_chunk_size": {
-                    "type": "integer",
-                    "default": 7500000,
-                    "description": "Limits the number of peptide_ids or predictions processed at once in the downstream processes and thus the memory footprint of these processes."
-                },
                 "prodigal_mode": {
                     "type": "string",
                     "default": "meta",
-                    "description": "Prodigal mode, 'meta' or 'single'."
+                    "description": "Prodigal mode, 'meta' or 'single'.",
+                    "fa_icon": "fas fa-cogs"
+                },
+                "memory_usage_log_deep": {
+                    "type": "boolean",
+                    "description": "Memory usage output \"deep\" for pandas script for more detailed calculation of memory usage (needs more resources).",
+                    "hidden": true,
+                    "fa_icon": "fas fa-align-justify"
+                },
+                "chunk_size": {
+                    "type": "integer",
+                    "default": 75000,
+                    "description": "Maximum chunk size (#peptides) for epitope prediction jobs is scaled up for downstream and upstream processes by chunk_size_scaling parameter.",
```

```suggestion
                    "description": "Maximum chunk size (#peptides) for epitope prediction jobs. Is scaled up by `chunk_size_scaling` parameter and used in downstream and upstream processes as well (e.g. for internal processing of peptide_id chunks in pandas scripts to reduce memory usage) .",
```
hmm...vielleicht so?

---

Yes that's way better, I didn't really know how to explain shortly but an example helps a lot.